openedx/openedx-events
# docs: [FC-0074] add docs about creating and consuming events

## Description

Update how-to documents to create and consume a new event considering design suggestions introduced in #438 and old docs as well.

Here are the main documents to review:
- How to Create a New Event with Long-Term Support: https://github.com/openedx/openedx-events/pull/439/files#diff-4ddc667d74faa397c775466201343c217838a6b92c089cff0fdd015d3d7918c9
- Consume an Event: https://github.com/openedx/openedx-events/pull/439/files#diff-bbbb0da0a033c988afb7eefca64da1f676a6b7c5d343904e77c865fd5f22cacf

The other changes are improvements to the current docs:
- Change continuous to imperative verb forms in titles
- Add assumptions for each how-to
- Complete reference for architecture subdomains

## Supporting information

Addresses #360 #362 

## Testing instructions

- You can review the docs here: https://docsopenedxorg--439.org.readthedocs.build/projects/openedx-events/en/439/. 
- It'd also be incredibly useful to follow the guide to verify the steps.

## Deadline

None

## Other information

Depends on feedback on #438 

## Checklists

Check off if complete *or* not applicable:

**Merge Checklist:**

**Post Merge:**
      finished.
